draft-ietf-quic-load-balancers.md

```diff
@@ -179,18 +179,18 @@ Subsequent sections of this document refer to the contents of this octet as the
 
 ## Config Rotation {#config-rotation}
 
-The first two bits of any connection-ID MUST encode the configuration phase of
-that ID.  QUIC-LB messages indicate the phase of the algorithm and parameters
+The first three bits of any connection-ID MUST encode the configuration phase
+of that ID.  QUIC-LB messages indicate the phase of the algorithm and parameters
 that they encode.
 
 A new configuration may change one or more parameters of the old configuration,
 or change the algorithm used.
 
 It is possible for servers to have mutually exclusive sets of supported
-algorithms, or for a transition from one algorithm to another to result in Fail
-Payloads.  The four states encoded in these two bits allow two mutually
-exclusive server pools to coexist, and for each of them to transition to a new
-set of parameters.
+algorithms, or for servers to not have the trust relationship necessary to
+share QUIC-LB keys or other sensitive material.  The eight states encoded in
+these three bits allow three mutually exclusive server pools to coexist, and for
```

Comment:
```suggestion
these three bits allow four mutually exclusive server pools to coexist, and for
```

That assumes of course that you need two keys per server pool, as opposed to rotating each pool one by one, in which case you can have seven (assuming excellent, though probably unrealistically difficult, coordination).

I don't think that you need to be so definitive about this.  Say that you might use this to create 4 distinct pools that are able to independently use two distinct configurations (and therefore keys), or other arrangements.

Reply:
Since '111' is reserved for 5-tuple routing, there are 7 codepoints. I believe 3 is the correct number. But I should say "at least" three.